pkg/tsdb/prometheus/client/client.go

```diff
@@ -42,7 +43,7 @@ func (c *Client) QueryRange(ctx context.Context, q *models.Query) (*http.Respons
 	qs.Set("end", formatTime(tr.End))
 	qs.Set("step", strconv.FormatFloat(tr.Step.Seconds(), 'f', -1, 64))
 
-	return c.fetch(ctx, c.method, u, qs)
+	return c.fetch(ctx, c.method, u, qs, nil)
```

Comment:
All this other uses of fetch are not in use now as they are used only behind streaming feature flag.

@toddtreece not sure though how it was supposed to work with POST request. Seems like it assumed it will just use `qs` and make a POST body out of it but if the request from front end was POST there would be no `qs` and the `body` would be lost, wouldn't it?

Reply:
just adding a note from our call since we discussed it there. this change will break streaming queries, but it can be fixed during a future HTTP client refactor